build.gradle

```diff
@@ -118,14 +118,14 @@ subprojects {
     apply plugin: 'com.google.protobuf'
     apply plugin: 'java'
 
-    apply plugin: 'org.spine3.tools.javadoc-checker'
+    apply plugin: 'org.spine3.tools.spine-codestyle-checker'
     // Javadoc checker plugin allows finding FQN links with the wrong format.
```

Comment:
This description should be updated as well.

In particular, we need to explain why this text is about links, but the plugin name is `checkCodestyle`. Or rewrite the text.